shell/platform/darwin/ios/framework/Source/FlutterEngine.mm

```diff
@@ -1027,8 +1028,13 @@ - (FlutterEngine*)spawnWithEntrypoint:(/*nullable*/ NSString*)entrypoint
   flutter::Shell::CreateCallback<flutter::Rasterizer> on_create_rasterizer =
       [](flutter::Shell& shell) { return std::make_unique<flutter::Rasterizer>(shell); };
 
-  std::unique_ptr<flutter::Shell> shell =
-      _shell->Spawn(std::move(configuration), on_create_platform_view, on_create_rasterizer);
+  std::string initial_route;
```

Comment:
The naming convention for objc is camel case.  It's confusing having `initialRoute` and `initial_route` in the same context.  I'm not sure if we have an established convention for this but how about `initialRoute` and `cppInitialRoute`?

Reply:
Done